addons/website/views/website_templates.xml

```diff
@@ -1757,17 +1771,16 @@
     </xpath>
 </template>
 
+<!-- Descriptive -->
 <template id="template_footer_descriptive" inherit_id="website.layout" name="Descriptive" active="False">
     <xpath expr="//div[@id='footer']" position="replace">
         <div id="footer" class="oe_structure oe_structure_solo" t-ignore="true" t-if="not no_footer">
-            <section class="s_title pt48 pb24" data-vcss="001" data-snippet="s_title">
-                <div class="container s_allow_columns">
```

Comment:
Can you explain why you removed 's_allow_columns' class? You're maybe right but I would want to know :)